### ISSUE TITLE
pywbemtools uses internal functionality of pywbem

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbemtools#489]
Pywbemtools uses the following internal functionality of pywbem:
```
$ grep "from pywbem\." . -r --include="*.py"
./pywbemtools/pywbemcli/_cmd_class.py:from pywbem._nocasedict import NocaseDict
./pywbemtools/pywbemcli/_association_shrub.py:from pywbem._utils import _to_unicode, _ensure_unicode, _format
./pywbemtools/pywbemcli/_association_shrub.py:from pywbem._nocasedict import NocaseDict
./pywbemtools/pywbemcli/_common.py:from pywbem.cim_obj import mofstr
./pywbemtools/pywbemcli/_common.py:from pywbem.cim_obj import NocaseDict
./pywbemtools/pywbemcli/_displaytree.py:from pywbem.cim_obj import NocaseDict
./tests/unit/test_common.py:from pywbem._nocasedict import NocaseDict
```

The use of `pywbem.cim_obj` is particularly bad because that module will be renamed in pywbem 1.0.0 (and this is how the issue surfaced).

**Comment by KSchopmeyer** [comment]
We need a proposal for each case since this causes problems with python 1.0

**Comment by KSchopmeyer** [comment]
1. _to_unicode - Used one place in a method within the AssociationShrub class that creates a folded wbem_url.  Since it is only a single line of code (return obj.decode("utf-8") it is easily replaced.

2. _ensure_unicode - Similar to 1 in that the pywbem function is very small and can be replaced with its code. Again this is used only within the AssociationShrub in the method that folds a wbem url. The pywbem code is:

    if obj is not None:
        obj = bool(obj)
    return obj

3. _format. Again used only it a single method within the AssociationShrub.  However, this is a more general issue in that it fills a general need for a more protected formatter that was done in pywbem for a specific reason but that we have not generalized into pywbemcli.

4. mofstr - Used in _common.py as part of value formatter for properties and parameters in the following methods
   a. _format_instances_as_rows - That prepares values of properties in instances for display as cells in a table.
   b. process_invode_method to prepare returned parameters for display.

**Comment by andy-maier** [comment]
Addressed in PR #515 for master and TBD for 0.5.1.
